components/brave_ads/browser/ad_notification.h

```diff
@@ -23,7 +23,8 @@ struct NotificationInfo;
 
 namespace brave_ads {
 
-const char* const kBraveAdsUrlPrefix ="chrome://brave_ads/?";
+// On Android, this URL must represent an HTTP or HTTPS web origin.
+const char* const kBraveAdsUrlPrefix = "https://www.brave.com/ads/?";
 
```

Comment:
This was required in C80, I believe due to this changeset:

https://chromium-review.googlesource.com/c/chromium/src/+/1886890